modules/rostests/apitests/dbghelp/pdb.c

```diff
@@ -57,6 +58,118 @@ static void deinit_sym()
     ((PSYMBOL_INFO)(buff))->MaxNameLen = MAX_SYM_NAME; \
 } while (0)
 
+/* modified copy of function from apitests/apphelp/apitest.c */
+BOOL get_module_version(
+    IN HMODULE mod,
```

Comment:
SAL?

Reply:
What is SAL standing for?

Reply:
https://reactos.org/wiki/Abbreviations#S

> SAL -- Source Annotation Language, Standard Annotation Language

Reply:
SAL ... I assume because of the IN / OUT.  I find it helpful - why not? Should i remove it?

Reply:
It's supposed to be `_In_`

Reply:
Ah, ok i will fix it. Thanks.